src/common/monitor-view-events.js

```diff
@@ -74,7 +74,7 @@ function handleRender() {
 // Monitor a view's state, propagating attach/detach events to children and firing dom:refresh
 // whenever a rendered view is attached or an attached view is rendered.
 function monitorViewEvents(view) {
-  if (view._areViewEventsMonitored) { return; }
+  if (view.monitorViewEvents === false || view._areViewEventsMonitored) { return; }
 
```

Comment:
I would swap the check order since areViewEventsMonitored will be the most common flag set